### PR TITLE
SASL_SSL why?

### PR DESCRIPTION
**Pull request by qiannianshaonian**
Invalid value "SASL_SSL" for configuration property "security.protocol"

### CODE REVIEW

**Comment by qiannianshaonian** [comment]
@edenhill Can you take a look?

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ func main() {
 		"bootstrap.servers": "localhost",
 		"group.id":          "myGroup",
 		"auto.offset.reset": "earliest",
+		"security.protocol": "SASL_SSL",
```
Comment (edenhill) [inline README.md]: This is environment-specific, some clusters require SASL, SSL, or both.
And this configuration is not complete with SASL_SSL, also needs to set mechanisms, usernames, kerberos stuff, etc.

The README examples are aimed to be minimal, any environment-specific configuration is outside their scope.

Comment (qiannianshaonian) [reply README.md]: I have set up the sasl.username,sasl.password sasl.mechanisms, security.protocol and so on locally.  the demo is  https://github.com/Azure/azure-event-hubs-for-kafka/blob/master/quickstart/go/consumer/consumer.go Can you take a look?

Comment (edenhill) [reply README.md]: Looks good, apart from the sasl.username $ConnectionString reference.

**Pull request closed without merging**